dbtool.py

```diff
@@ -480,13 +491,9 @@ def exporttestdata(db: DbContext):
         oca_ssh_host=args.oca_ssh_host,
         oca_ssh_user=args.oca_ssh_user,
         oca_ssh_pkey=args.oca_ssh_pkey,
-        is_testing=args.use_test_data,
+        is_testing=False if cmd == "exporttestdata" else args.use_test_data,
```

Comment:
just for my own clarification, why this exception?

Reply:
Existing documentation says that the exporttestdata requires a fully loaded version of the database (I think because we have some specific BBLs hardcoded into that function and our tests, and so it needs to have them all available). 

And so there isn't an argument for using test data on that command within dbtool.py, so I just put this exception in so that doesn't fail when it's missing. 

Maybe it'd be better to break up the if/else block for commands - putting exporttestdata case first, then set up this oca_config, then run through the other command cases where that argument will be present.

Reply:
does args.use_test_data default to true? maybe we could have something where if it's none/undefined, we set it to false here?

Reply:
After looking more closely at this I realized that `use_test_data` argument wasn't actually being used, since the `builddb` command is always the full data, and `loadtestdata` is always test data. So I now only create the `oca_config` when it's one of these commands, and it just assigns `use_test_data` appropriately and the argument is removed from the CLI.